t/subname.t

```diff
@@ -82,12 +82,13 @@ is($x->(), "main::foo");
 {
   {
     package Blarf;
-    sub gorp { (caller(0))[3] }
+    sub gorp { 1 }
   }
   my $sub = \&Blarf::gorp;
   delete $::{'Blarf::'};
-  my $name = $sub->();
-  is subname($sub), $name,
+  my $package = $] < 5.010 ? 'Blarf' : '__ANON__';
```

Comment:
Zefram was recommending that all uses of $] should use string interpolation to avoid floating point issues: e.g. `my $package = "$]" < '5.010' ..`